src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,507 @@
+      description: |
+        This represents the id of the policy.
+      example: 'tpr12345xyz'
+    description:
+      allOf:
+        - $ref: ./fields.yaml#/description
+      description: |
+        This represents the description of the policy.
+      example: Receipt is mandatory for expense above $50
+    source:
+      $ref: ./fields.yaml#/policy_source
+    expense_limit_type:
+      $ref: ./fields.yaml#/policy_limit_type
+    expense_currency:
+      $ref: ./fields.yaml#/currency
+    expense_amount_max:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      example: 10000
+    expense_amount_min:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      example: 500
+    expense_distance_max:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      example: 10000
+    expense_distance_min:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      example: 500
+    expense_category_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [2222, 1221]
+      description: |
+          List of category ids
+    expense_category_ids_op:
+      $ref: ./fields.yaml#/policy_operator
+    expense_project_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [2222, 1221]
+      description: |
+          List of category ids
+    expense_project_ids_op:
+      $ref: ./fields.yaml#/policy_operator
+    expense_cost_center_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [2222, 1221]
+      description: |
+          List of category ids
+    expense_cost_center_ids_op:
+      $ref: ./fields.yaml#/policy_operator
+    expense_exchange_rate_difference_percentage_allowed:
+      type: number
+      example: 10
+    expense_distance_difference_percentage_allowed:
+      type: number
+      example: 10
+    expense_custom_fields:
+      $ref: './fields.yaml#/custom_fields'
+    expense_custom_fields_op:
+      $ref: ./fields.yaml#/policy_operator
+    expense_locations:
+      description: |
+        This field will be present for travel related categories.
+        For travel between Place A to Place B, this array will contain two items, From location at array index 0 and To location at array index 1.
+      type: array
+      items:
+        $ref: './fields.yaml#/location'
+    expense_locations_op:
+      $ref: ./fields.yaml#/policy_operator
+    expense_creation_delay_days_max:
+      type: number
+      example: 12
+    expense_on_holiday_allowed:
+      type: boolean
+      example: True
+    expense_without_receipt_allowed:
+      type: boolean
+      example: True
+    expense_spent_at_max:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+    expense_spent_at_min:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+    expense_payment_modes:
+      type: array
+      description: |
+        Payment mode for expense
+      items:
+        type: string
+      example:
```

Comment:
check if this field already exists ?
if not then create and use; use enum inside